lte/gateway/c/oai/tasks/amf/amf_security_mode_control.cpp

```diff
@@ -221,36 +225,37 @@ static int security_mode_t3560_handler(zloop_t* loop, int timer_id, void* arg) {
         LOG_AMF_APP, "T3560 timer   timer id %d ue id %d\n", smc_proc->T3560.id,
```

Comment:
this statement is providing sufficient info for troubleshooting. other statements above can also follow this approach.